### ISSUE TITLE
Label anything：Can I use other pre-trained models？

### ISSUE THREAD

**Issue by Rongke-Wei** [open-mmlab/playground#106]
How to obtain the pre-trained model of SAM。Can we input the pretrained model of SAM trained on someone dataset.

**Comment by JimmyMa99** [comment]
The SAM weights we use are officially provided, you can download and load the corresponding weights according to the links and methods in the document. Regarding the issue of using SAMs trained on partial data, it seems that there is no official training code provided, but if you have the corresponding weights, you can also load them by referring to the code provided.
